projects/angular/src/popover/signpost/_signposts.clarity.scss

```diff
@@ -20,7 +22,14 @@
     }
 
     .signpost-action {
-      min-width: variables.$clr_baselineRem_1;
+      // to be removed once shim is fully integrated.
```

Comment:
You can update the comments with the new tag approach ( ATM ) that you proposed.

Reply:
Good catch I forgot about it.
Fixed in commit: d5631b6